Make adapter info into `frontMatter`-powered snippet table (#2040)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/no-configs.md

---
title: "No specifc configurations for this Adapter"
id: "no-configs"
---

> If you came here from a data platform setup page, it means there either is:

Reviewer: ```suggestion
If you were guided to this page from a data platform setup article, it most likely means:
```

> - nothing an end-user needs to know about using dbt on a given data platform beyond setting up the profile, or

Reviewer: ```suggestion
- Setting up the profile is the only action the end-user needs to take on the data platform, or
```

> - there is but it is currently undocumented

Reviewer: ```suggestion
- The subsequent actions the end-user needs to take are not currently documented
```
